libafl/src/monitors/mod.rs

```diff
@@ -23,6 +23,9 @@ pub use tui::TuiMonitor;
 #[cfg(all(feature = "prometheus_monitor", feature = "std"))]
 pub mod prometheus;
 
+#[cfg(all(feature = "statsd_monitor", feature = "std"))]
```

Comment:
statsd_monitor implies std, so there's no need to specify both here (same is true for the prometheus monitor above, if you want to change it)